tests/system_tests_expandvars.py

```diff
@@ -0,0 +1,91 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import os
+import sys
+import subprocess
+from subprocess import STDOUT, CalledProcessError
+import shutil
+from system_test import TestCase
+
+env_config = {
+    'PORT': '12345',
+    'NAME': 'YoYoMa'
+}
+
+current_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
It should be possible to use `import pathlib`, it is present since Python 3.4 and it is somewhat nicer to use than `os.path`. But that is matter of preference, of course. https://docs.python.org/3/library/pathlib.html

Reply:
I looked into pathlib and to me the os.path approach looks to be more simpler.

Reply:
ok, np